src/components/simulator.js

```diff
@@ -29,6 +29,7 @@ export const useSimulator = () => {
     const error = useRef("");
     const [exitCode, setExitCode] = useState("");
     const [code, setCode] = useState("");
+    const history = useRef([]);
```

Comment:
Forgot I did this (was trying to fix a bug in an old approach). Gonna move it back to Console.jsx